db/fixtures/ae_datastore/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/used.rb

```diff
@@ -0,0 +1,15 @@
+#
+# Description: calculate entity used quota values
+#
+
+def consumption(source)
+  {
+    :cpu                 => source.allocated_vcpu,
+    :memory              => source.allocated_memory / 1024**2,
```

Comment:
@tinaafitz 
For clarity would it make sense to convert the division of 1024**2 to call a function like
bytes_to_megabytes(source.allocated_memory)
This would tell us what is the source type and what is it being converted to.